src/librbd/io/AioCompletion.cc

```diff
@@ -237,8 +232,8 @@ void AioCompletion::complete_request(ssize_t r)
 
 bool AioCompletion::is_complete() {
   tracepoint(librbd, aio_is_complete_enter, this);
-  bool done = (this->state == AIO_STATE_COMPLETE);
-  tracepoint(librbd, aio_is_complete_exit, done);
+  bool done = completed.load(std::memory_order_acquire);
+  tracepoint(librbd, aio_is_complete_exit, completed);
```

Comment:
Let's match the exit tracepoint and the return value, as it was before:
```suggestion
  tracepoint(librbd, aio_is_complete_exit, done);
```